### PR TITLE
Automation fix for all services.

### PR DESCRIPTION
**Pull request by divyagovindaiah**
this pr contains

1. Hudi Configuration Issues: The checkpointing path is not configured.
2. Resource Configuration: CPU and memory resources for all services are configured in the global-resource-values.yaml.
3. Hudi Installation: Ensure that Hudi is included as part of the package installation
4. Kafka Configuration: Parameterize the Kafka partition and replication factor values.
5. Grafana Removal: Grafana is part of the kube-prometheus-stack Helm chart, so we removed Grafana from install.sh

### CODE REVIEW

File: helmcharts/kitchen/install.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ migrations)
 monitoring)
     rm -rf monitoring
     cp -rf ../obsrv monitoring
-    cp -rf ../services/{promtail,loki,grafana,kube-prometheus-stack,kafka-message-exporter,alert-rules} monitoring/charts/
+    cp -rf ../services/{promtail,loki,kube-prometheus-stack,kafka-message-exporter,alert-rules} monitoring/charts/
```
Comment (manjudr) [inline helmcharts/kitchen/install.sh:52]: why is grafana removed?

File: helmcharts/services/kube-prometheus-stack/charts/grafana/values.yaml [hunk 2]
```diff
@@ -1399,4 +1399,4 @@ extraObjects: []
 #
 # Alternatively, if you wish to allow secret values to be exposed in the rendered grafana.ini configmap,
 # you can disable this check by setting assertNoLeakedSecrets to false.
-assertNoLeakedSecrets: true
+assertNoLeakedSecrets: false
```
Comment (manjudr) [inline helmcharts/services/kube-prometheus-stack/charts/grafana/values.yaml:1402]: Is there any reason to turn it off?

**Review by manjudr — DISMISSED** [review]

File: helmcharts/kitchen/install.sh (hunk 1 above)
Comment (divyagovindaiah) [reply helmcharts/kitchen/install.sh:52]: its part of  kube-promethous-stack helm chart

File: helmcharts/services/kube-prometheus-stack/charts/grafana/values.yaml (hunk 2 above)
Comment (divyagovindaiah) [reply helmcharts/services/kube-prometheus-stack/charts/grafana/values.yaml:1402]: yeah  throwing error `Sensitive key 'smtp.password' should not be defined explicitly in values `during deployment thats why i removed

**Review by manjudr — APPROVED** [review]

**Pull request merged**